bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,110 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "mozorg/home/home.html" %}
+
+{% add_lang_files "mozorg/home/index-2016" "firefox/features/index" "firefox/sync" "firefox/whatsnew_38" "firefox/whatsnew_50" %}
```

Comment:
Ugh, this is really bad…

Any of these files could activate the home page.

Reply:
> Ugh, this is really bad…

Is this technically bad, or just messy?

> Any of these files could activate the home page.

Not sure what this means.

Reply:
Bedrock shows a template on locale X based on the fact that the lang file is activated (has `##active##` at the beginning).

Here you are importing a ton of lang files for one template. Result: if even one of them is complete and activated, the entire template is active, even if we don't have all the strings translated.

Take `firefox/sync.lang`, is a template as old as it can be, with 89 languages activate. 

Now set your local install as PROD, and load the home page for 'ach': it should load almost completely in English, instead of redirecting to en-US.